landoui/treestatus.py

```diff
@@ -211,7 +211,17 @@ def treestatus_tree(tree: str):
     """Display the log of statuses for an individual tree."""
     api = TreestatusAPI.from_environment()
 
-    logs_response = api.request("GET", f"trees/{tree}/logs")
+    try:
+        logs_response = api.request("GET", f"trees/{tree}/logs")
+    except LandoAPIError as exc:
+        if not exc.detail or not exc.status_code:
+            raise
+
+        error = f"Error received from LandoAPI: {exc.detail}"
+        logger.error(error)
+        flash(error, error)
```

Comment:
This should be `flash(error, "error")`. The second argument is the category.

Reply:
Good catch, thanks.